dev-cpp/gtest-parallel/gtest-parallel-20220621.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{13..14} )
```

Comment:
We may still have users staying on python3_11/12 rather than the default, so I'd keep these for now assuming tests pass.